core/services/flux_monitor_test.go

```diff
@@ -251,7 +265,8 @@ func TestPollingDeviationChecker_StartStop(t *testing.T) {
 	// Start() with no delay to speed up test and polling.
 	done := make(chan struct{})
 	go func() {
-		checker.Start(context.Background(), ethClient) // Start() polling
+		checker.Connect(ethClient)
```

Comment:
Is this mocked? If so, is there any point in connecting here?

Reply:
Its mocked out but instantiates fields within the struct.  Start kind of assumes Connect wast run.

Reply:
I'll clarify this in the code